ts/features/itwallet/machine/eid/guards.ts

```diff
@@ -29,7 +29,10 @@ export const createEidIssuanceGuardsImplementation = (
    * opened with io-react-native-login-utils was closed by the user.
    */
   isNativeAuthSessionClosed: ({ event }: { event: EidIssuanceEvents }) => {
```

Comment:
When I close the native webview I got this error:
```
{
  "code": "ERR_IO_WALLET_AUTHORIZATION_ERROR",
  "name": "AuthorizationError"
}
```
The pipe returns `false` and instead to return back to the method selection we go to the failure screen.